platform/linux-generic/odp_packet_io.c

```diff
@@ -967,7 +1006,22 @@ static inline int packet_vector_send(odp_pktout_queue_t pktout_queue, odp_event_
 		ODP_ASSERT(entry != NULL);
 
 		odp_atomic_add_u64(&entry->s.stats_extra.out_discards, discards);
-		odp_packet_free_multi(&pkt_tbl[sent], discards);
+
+		if (odp_unlikely(_odp_pktio_tx_compl_enabled(entry)))
+			for (i = sent; i < num; i++) {
```

Comment:
If-statement has multiple lines, but misses bracets {}.  Code is more readable and robust to maintain, when for and if-else statements have their own bracets.

Reply:
Moved to own function in v2.